roaringbitmap/src/main/java/org/roaringbitmap/RoaringArray.java

```diff
@@ -264,12 +273,13 @@ protected void copyRange(int begin, int end, int newBegin) {
   }
 
   /**
-   * Deserialize.
+   * Deserialize. If the DataInput is available as a byte[] or a ByteBuffer, you could prefer
+   * relying on {@link #deserialize(ByteBuffer)}. If the InputStream is &gt;= 8kB, you could prefer
```

Comment:
Is that &gt;= or &lt;=?

Reply:
gt in &gt; stands for 'Greater Than'. Javadoc complained about '>'.